gmso/lib/jsons/.gitignore

```diff
@@ -0,0 +1 @@
+TestTemplate.json
```

Comment:
Is there a reason to have this file here and not at the root of the repo?

Reply:
It would not be necessary to add .gitignore if we don't provide a python script to save a file which I think we shouldn't. This was added because there a test that adds the file without deleting it. So, as @justinGilmer suggested, I will remove the file(from the test after completion).